.circleci/config.yml

```diff
@@ -1139,16 +1155,19 @@ workflows:
             - assets_precompile
           <<: *only-master-filter
 
-  portafly_build:
+  portafly:
     jobs:
       - manual_approval:
           type: approval
       - notify_start:
           requires:
             - manual_approval
-      - portafly:
+      - portafly-tests:
```

Comment:
You should follow convention and align with the rest of the tasks and use underscores, snake_cased names.

Reply:
I don't know if there's such convention whatsoever 😂 I've followed the most common format in the config file: 51 kebab vs 21 snake

Reply:
I agree that most artifacts are kebab, but the name of the workflows/jobs are snake: `mysql_build`, `postgres_build`, `oracle_build`, `assets_precompile`, `dependencies_npm`etc...